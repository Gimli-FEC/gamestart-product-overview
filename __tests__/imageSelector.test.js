import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';
import $ from 'jquery';

import ImageSelector from '../client/components/imgSelector.jsx';

jest.mock('$');

afterEach(cleanup);

test('renders', () => {
  const { asFragment } = render(<ImageSelector />);
  expect(asFragment()).toMatchSnapshot();
});