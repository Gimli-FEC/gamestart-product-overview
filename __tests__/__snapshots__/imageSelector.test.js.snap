// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="imgSelector__Grid-yx4597-1 gdALxi"
  >
    <div
      class="imgSelector__Row-yx4597-2 kPRbKz"
    >
      <div
        class="imgSelector__Col-yx4597-3 fvCZmB"
      >
        <div
          class="imgSelector__ThumbnailEnd-yx4597-4 kgGYTw"
        >
          SEE MORE
        </div>
      </div>
      <div
        class="imgSelector__Col-yx4597-3 diidOI"
        size="5"
      >
        <div
          class="mainImage__MainContainer-zvt6z9-0 kaSyTb"
          style="background-position: 0% 0%;"
        >
          <img
            class="mainImage__Image-zvt6z9-1 ccjQdL"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
